Sources/Thumbprint/Presentations/Sheet/PartialSheetPresentation.swift

```diff
@@ -76,7 +80,9 @@ open class PartialSheetPresentation: NSObject, UIViewControllerTransitioningDele
             return nil
         }
 
-        return PercentDrivenInteractiveTransition(panGestureRecognizer: partialSheetPresentationController.panGestureRecognizer)
+        return PercentDrivenInteractiveTransition(panGestureRecognizer: partialSheetPresentationController.panGestureRecognizer) { didFinish in
+            partialSheetPresentationController.didCompleteInteractiveTransition(success: didFinish)
```

Comment:
i find both `didFinish` and `success` to be a bit confusing for names here. Maybe `didDismiss` ?

Reply:
Frankly, the API is a little confusing.  The dismiss is what triggers the interactive animation, so technically, it did "dismiss" even if the user doesn't swipe all the way and cancels the interaction.  The question is, was the dismissal able to complete or not.

I could refactor this to not trigger the callback in the cancelled case.  Maybe that would be clearer.

Reply:
I see. That makes sense then. 

I'll leave it up to you if you want to change the api.